repos/fdbt-site/src/pages/salesConfirmation.tsx

```diff
@@ -121,33 +121,45 @@ const SalesConfirmation = ({
     fareType,
     hasCaps,
     selectedCap,
-}: SalesConfirmationProps): ReactElement => (
-    <TwoThirdsLayout title={title} description={description} errors={[]}>
-        <CsrfForm action="/api/salesConfirmation" method="post" csrfToken={csrfToken}>
-            <>
-                <h1 className="govuk-heading-l">Check your sales information answers before submitting</h1>
-                <ConfirmationTable
-                    header="Sales Information"
-                    confirmationElements={buildSalesConfirmationElements(
-                        salesOfferPackages,
-                        startDate,
-                        endDate,
-                        fareType,
-                        hasCaps,
-                        selectedCap,
-                    )}
-                />
-                <h2 className="govuk-heading-m">Now submit your data to create the product</h2>
+}: SalesConfirmationProps): ReactElement => {
+    const [isSubmitting, setIsSubmitting] = useState(false);
+    return (
+        <TwoThirdsLayout title={title} description={description} errors={[]}>
+            <CsrfForm action="/api/salesConfirmation" method="post" csrfToken={csrfToken}>
+                <>
+                    <h1 className="govuk-heading-l">Check your sales information answers before submitting</h1>
+                    <ConfirmationTable
+                        header="Sales Information"
+                        confirmationElements={buildSalesConfirmationElements(
+                            salesOfferPackages,
+                            startDate,
+                            endDate,
+                            fareType,
+                            hasCaps,
+                            selectedCap,
+                        )}
+                    />
+                    <h2 className="govuk-heading-m">Now submit your data to create the product</h2>
 
-                <p className="govuk-body">
-                    By submitting this data you are confirming that, to the best of your knowledge, the details you are
-                    providing are correct.
-                </p>
-                <input type="submit" value="Accept and submit" id="continue-button" className="govuk-button" />
-            </>
-        </CsrfForm>
-    </TwoThirdsLayout>
-);
+                    <p className="govuk-body">
+                        By submitting this data you are confirming that, to the best of your knowledge, the details you
+                        are providing are correct.
+                    </p>
+                    <input
+                        type="submit"
+                        value="Accept and submit"
+                        id="continue-button"
+                        className="govuk-button"
+                        disabled={isSubmitting}
```

Comment:
are we restricting for the sales confirmation ... is it like two products are created?

Reply:
Exactly yeah